contributions/covid19/tommysam-jii/COVID19_COURSE_INFO.md

```diff
@@ -0,0 +1,104 @@
+
+Today, our Jenkins server is running locally on Johanna's computer. To get access to our Jenkins server, you need to go to [92.34.7.239:8080](92.34.7.239:8080) and log in with these credentials:
+
+Username: test_user  
+Password: Test2482
+
+The server might not be up and running all the time, but down below we have a screencast of our Jenkins server in action. If the server is not up and running and you want to test our pipeline, please contact Johanna at jii@kth.se.
+
+Click on the image below to be re-directed to the screencast at Youtube:
+
+[![](/images/jenkins_screencast.png)](https://youtu.be/XFCERzBMqZc) 
+
+The video has been cut at some points to shorten the length of the video and remove parts that does not add any new information to the filmed screencast. 
+
+#### Firebase
+Firebase is a serverless computing service, which provides a real-time database for it's users. Firebase provides an API that allows data to be synchronized across all the clients. The application data is stored on Firebase's own cloud.[[1]](https://en.wikipedia.org/wiki/Firebase) Serverless computing is relevant for DevOps, since it shortens the release cycle. By using a serverless architecture, the developers can focus on the functionality of the application and the application's GUI, instead of writing scripts for the server or the database. 
+ 
+<a name="crit"> </a>
+
+### Criteria for COVID19 Assignment
+
+We aim to achieve at least these criteria:
+
+**The contribution is related to COVID-19**
+* A mobile and web application that helps KTH students stay updated on changes in examination due to COVID-19.
+
+**The contribution is related to / built with DevOps technology**
+* We have created our own Jenkins server, and built a small pipeline that is connected to our private repository for the application. We are also using Firebase in this project, which is a serverless computing service.
+
+**The contribution is novel**
+* We haven't found any application that is similar to ours yet. 
+
+**The contribution is helpful**
+* If the app uses properly, it is a good way for the KTH students to get an overview on how the courses has changed due to COVID19 and how the examination will take form. The examiners might use different channels to spread the information, and as a student it might be difficult to find that information because of that. By using this application, we can gather all information at one place and update the application as soon as something has changed. 
+There a lot of courses at KTH, and some of them might have found a good solution to examine a course that usually has an exam that takes place in school. This application could also be used for examinars to see how other courses are handling the situation at get some inspiration.
+
+**The contribution attracts support from other KTH students**
```

Comment:
Support means using the app not reading the code.

What's your dissemination plan? In which channels did/will you announce it?